src/apps/weblib/cinterface.cpp

```diff
@@ -94,7 +96,11 @@ extern const char *vizzu_version()
 	return Interface::getInstance().version();
 }
 
-void vizzu_setLogging(bool enable) { Interface::setLogging(enable); }
+void vizzu_setLogging(bool enable)
+{
+	Interface::setLogging(enable);
+	IO::log() << "Logging enabled: " << enable;
```

Comment:
This is weird. When logging disabled, this line will not print. 
I think when setLogging(false) called, the log need to print before Interface::setLogging.

Another note: I think this should be not part of C-interface but interface, or registered callback as a logging enable-change function